src/gui/plugins/align_tool/AlignTool.hh

```diff
@@ -114,14 +114,14 @@ namespace gazebo
     /// \param[in] _state New state to add by enum AlignState
     public: void AddState(const AlignState &_state);
 
-    /// \brief Makes the node transparent
-    /// \param[in] _node The node to make transparent
-    public: void MakeTransparent(const rendering::NodePtr &_node);
-
-    /// \brief Reset the node to its solid form, intended to be called
-    /// after MakeTransparent
-    /// \param[in] _node The node to make solid
-    public: void MakeSolid(const rendering::NodePtr &_node);
+    /// \brief Updates the node to increase its transparency or reset
+    /// back to it's original transparency value, an opaque call requires
+    /// a previous transparent call
+    /// \param[in] _node The node to update
+    /// \param[in] _makeOpaque true if updating to transparent, false to
```

Comment:
true if updating to **opaque**?

Reply:
Updated the bool to actually be `makeTransparent` in [d3b89fa4](https://github.com/ignitionrobotics/ign-gazebo/pull/59/commits/d3b89fa49cc6c469198205e7cd3b335e00c68bfe)